com.github.scrivano.scrivano.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.github.scrivano.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
this is a very very broad file system permission

Reply:
I've now set up more specific file system and sandbox permissions